napari/utils/events/containers/_selection.py

```diff
@@ -10,9 +10,16 @@
 
 
 class Selection(EventedSet[_T]):
-    """An unordered collection of selected elements, with a `current` item.
+    """An model of selected items, with a `active` and `current` item.
```

Comment:
@jni might want to look here

Reply:
Clearly from an old PR, but @tlambert03, is the convention for `_T` rather than `T` widespread? I don't really see the need for the underscore... But this is just a side note, don't worry about it. =)

Reply:
I'm sure I started doing that from examples in the standard lib... but I don't have strong feelings about it.  you see both